### DISCUSSION TITLE
ScrollArea flickering

### DISCUSSION THREAD

**Discussion by erwinstone** [mantinedev/mantine#3035] [Q&A]
https://user-images.githubusercontent.com/96282452/203497508-a0e41146-3e64-4a0f-a160-497fea23e88a.mp4

I don't know if this only happens on my device or what

**Comment by noahsb** [comment]
According to the [docs](https://mantine.dev/core/scroll-area/#usage):

> hover – scrollbars are visible on hover

Because you are hovering on/off the component, the scrollbar is becoming visible/hidden every time this occurs. That is the expected behavior for scrollbars with `type="hover"`

If you don't like this behavior, you can choose a different type from the docs

**Reply by erwinstone** [reply]
yeah, but it comes off the top first like it's lagging
